spec/views/posts/post_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Renders the post show page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Baraka Danny', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Am a ruby developer', post_counter: 9)
+    @first_post = Post.create(title: 'Hello', text: 'This is a post', comments_counter: 0, likes_counter: 0,
+                              user_id: @user.id)
+    5.times do |_i|
+      Comment.create(text: 'Interesting topic', user_id: @user.id, post_id: @first_post.id)
+    end
+    visit user_post_path(@first_post.user, @first_post)
+  end
+
+  scenario 'displays the post title' do
+    expect(page).to have_content(@first_post.title)
+  end
+
+  scenario 'disolays number of comments' do
+    expect(page).to have_content('5')
+  end
+
+  scenario 'displays number of likes' do
+    expect(page).to have_content('0')
+  end
+
+  scenario 'displays the content of the post' do
```

Comment:
- [ ] please make sure to write tests for all the below senario's:
- Tests that asserts you can see the post's title.
- Tests that asserts you can see who wrote the post.
- Tests that asserts you can see how many comments it has.
- Tests that asserts you can see how many likes it has.
- Tests that asserts you can see the post body.
- Tests that asserts you can see the username of each commentor.
- Tests that asserts you can see the comment each commentor left.

Reply:
@mikethreels 
Thank you for the review
Note: some of the tests have been made `(Post_show_spect)`, 
I made the requested changes

Reply:
well resolved! ✔️